diff --git a/dist/utils/instantiate.js b/dist/utils/instantiate.js
index 33cdff152e8b23d6853eb475097bda8c02712e01..0fa0568671878c68101fc95a4c9b096b0dbd8e2b 100644
--- a/dist/utils/instantiate.js
+++ b/dist/utils/instantiate.js
@@ -34,7 +34,7 @@ function instantiate(name, circuitConfig) {
         pubs: circuitConfig.pubs || [],
         params: circuitConfig.params || [],
     });
-    const targetDir = `./circuits/${directory}`;
+    const targetDir = `./circom/${directory}`;
     if (!(0, fs_1.existsSync)(targetDir)) {
         (0, fs_1.mkdirSync)(targetDir, {
             recursive: true,
